api/lib/infrastructure/repositories/certification-result-repository.js

```diff
@@ -34,7 +34,21 @@ module.exports = {
       .orderBy('certification-courses.lastName', 'ASC')
       .orderBy('certification-courses.firstName', 'ASC');
 
-    return certificationResultDTOs.map(_toDomain);
+    let complementaryCertificationCourseResultsByCertificationCourseId = [];
+    if (!isEmpty(certificationResultDTOs)) {
+      complementaryCertificationCourseResultsByCertificationCourseId =
+        await _selectComplementaryCertificationCourseResultsBySessionId({
+          sessionId: certificationResultDTOs[0]?.sessionId,
```

Comment:
```suggestion
          sessionId: certificationResultDTOs[0].sessionId,
```
On a fait la vérif juste au dessus que ce ne soit pas une collection vide, ça me paraît être une garde de trop

Reply:
En effet, c'est modifié.